sdk/cosmosdb/cosmos/src/globalEndpointManager.ts

```diff
@@ -228,7 +231,9 @@ export class GlobalEndpointManager {
       this.writeableLocations = resourceResponse.resource.writableLocations;
       this.readableLocations = resourceResponse.resource.readableLocations;
       this.enableMultipleWriteLocations = resourceResponse.resource.enableMultipleWritableLocations;
-      this.refreshPPAFFeatureFlag(resourceResponse.resource.enablePerPartitionFailover);
+      if (this.enablePartitionLevelFailover) {
```

Comment:
this is enabled by deafult, so should we rename the connection policy flag to disable PPAF?